Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:buster
+
+RUN apt-get update \
+ && apt-get install --yes \
+      wget \
+      python3-distutils \
+      python3-scapy \
+      vim \
```

Comment:
Production containers should have a minimum of packages, only those required to run. `vim` is not a runtime requirement, and this container is no longer a development environment. You can make a dev-env container with vim in it that is `FROM` this container, but `vim` does not go in the production container.

Reply:
Ok, fair point. Removing vim from here, but I will keep pciutils since that is proving useful in seeing which Mellanox ports on headnode are available within the container for trex to use.